client/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es5",
+        "noImplicitAny": false,
```

Comment:
`noImplicitReturns`, `noImplicitThis`, `noUnusedLocals`, `noUnusedParameters`, and `strictNullChecks`